src/components/Login/LoginInputs.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import classes from "./LoginInputs.module.css";
+
+const LoginInput = React.forwardRef((props, ref) => {
+  return (
+    <>
+      <div
+        className={`${classes.control} ${
+          props.isValid === false ? classes.invalid : ""

```

Comment:
There is a useful library that can help us render conditional classes. It's called `classnames`, check it out: https://www.npmjs.com/package/classnames